target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -74,7 +74,8 @@ ramips_setup_interfaces()
 	elecom,wrc-2533gst|\
 	iodata,wn-ax1167gr|\
 	iodata,wn-gx300gr|\
-	iodata,wnpr2600g)
+	iodata,wnpr2600g|\
+	iptime,a8004t)
```

Comment:
The code below implies a reversed order of LAN port labels compared to the internal switch. Is that correct?

Reply:
Yes it is correct. It has reverse ordered LAN port.